independent-publisher-connectors/Rijksmuseum/apiDefinition.swagger.json

```diff
@@ -0,0 +1,836 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Rijksmuseum",
+    "description": "Retrieve metadata of objects from the Rijksmuseum collection",
+    "version": "1.0",
+    "contact": {
+      "name": "Ashwin Ganesh Kumar",
+      "url": "https://www.sogeti.nl/",
+      "email": "ashwin.ganesh-kumar@sogeti.com"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://data.rijksmuseum.nl/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.rijksmuseum.nl/en/footer/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Data"
+    }
+  ],
+  "host": "www.rijksmuseum.nl",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/{culture}/collection/": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
```

Comment:
@ashes2076 May I ask you please to give a proper description for the response? for example you can use "OK", "Success", "Bad Request", etc. 
Please fix all responses in the swagger file.

Reply:
Yes I Have since fixed the responses

Reply:
Updated Both